src/helpers/queries.js

```diff
@@ -53,3 +53,11 @@ export const getOnboardingStatus = {
     fetchPolicy: fetchPolicies.noFetch()
   }
 }
+
+export const getContactByIds = (ids = []) => ({
+  definition: () => Q(CONTACTS_DOCTYPE).getByIds(ids),
+  options: {
+    as: `getContactByIds/${ids.join('')}`,
```

Comment:
:question: How does this work if there is no seperator between id strings?

Reply:
The IDs will be stuck together, which is fine in this case